templates/sidebar/roll.html

```diff
@@ -43,8 +43,8 @@ <h2>{{title}}</h2>
             {{/if}}
           {{/if_eq}}
           <tr>
-            <td>{{#if_eq rollType 'armor'}}{{localize "YZECORIOLIS.ArmorRating"}}: {{bonus}}{{else}}{{attributeName}}: {{attribute}}{{/if_eq}}</td>
```

Comment:
I'm not following what this change is fixing. Mind clarifying what was displaying incorrectly?

Reply:
That was a strange one.

If you roll on armor via a macro it showed an skill (which doesn't exist with armor-rolls). Outside of macro-rolls it wasn't displayed (probably why it wasn't detected till now).

![image](https://user-images.githubusercontent.com/36819852/199551633-bb6feab7-8c1c-42da-98c3-8ebd2e3fa70b.png)